mindmaps-test/src/test/java/io/mindmaps/test/graql/reasoner/ReasonerTest.java

```diff
@@ -188,6 +190,97 @@ public void testResourceAsVar5(){
     }
 
     @Test
+    public void testNoRelationType(){
+        MindmapsGraph lgraph = GeoGraph.getGraph();
+        String queryString = "match $x isa city;$y isa country;($x, $y);$y id 'Poland';";
+        String queryString2 = "match $x isa city;$y isa country;" +
+                    "(geo-entity: $x, entity-location: $y) isa is-located-in;$y id 'Poland';";
+        MatchQuery query = new Query(queryString, lgraph);
+        MatchQuery query2 = new Query(queryString2, lgraph);
+
+        Reasoner reasoner = new Reasoner(lgraph);
+        assertEquals(reasoner.resolve(query), reasoner.resolve(query2));
+    }
+
+    @Test
+    public void testNoRelationTypeWithRoles(){
+        MindmapsGraph lgraph = GeoGraph.getGraph();
+        String queryString = "match $x isa city;$y isa country;(geo-entity: $x, $y);$y id 'Poland';";
+        String queryString2 = "match $x isa city;$y isa country;" +
+                    "(geo-entity: $x, entity-location: $y) isa is-located-in;$y id 'Poland';";
+        MatchQuery query = new Query(queryString, lgraph);
+        MatchQuery query2 = new Query(queryString2, lgraph);
+
+        Reasoner reasoner = new Reasoner(lgraph);
+        assertEquals(reasoner.resolve(query), reasoner.resolve(query2));
+    }
+
+    @Test
+    public void testNoRelationTypeWithRoles2(){
+        MindmapsGraph lgraph = GeoGraph.getGraph();
+        String queryString = "match $x isa city;$y isa country;(geo-entity: $x, $y);";
+        String queryString2 = "match $x isa city;$y isa country;" +
+                "(geo-entity: $x, entity-location: $y) isa is-located-in;";
+        MatchQuery query = new Query(queryString, lgraph);
+        MatchQuery query2 = new Query(queryString2, lgraph);
+
+        Reasoner reasoner = new Reasoner(lgraph);
+        assertEquals(reasoner.resolve(query), reasoner.resolve(query2));
+    }
+
+    @Test
+    public void testAKO(){
+        MindmapsGraph lgraph = GeoGraph.getGraph();
+        String queryString = "match $x isa $type;$type ako geoObject;" +
```

Comment:
We just changed "ako" to "sub" so we can't merge until this is changed here sadly.